### DISCUSSION TITLE
[5.1.2] Clean installation without error but image is not showing...

### DISCUSSION THREAD

**Discussion by vuongquoc** [LycheeOrg/Lychee#2262] [Q&A] [answered]
Dears,
I've just make a clean installation for V5.1.2
Everything working just fine but when i upload picture, it doesn't appear...
Just an icone like a broken link but uploading window said it was successfully uploaded...
Does anyone like me?

Please have a look...
https://bachda.xyz/public/

Thank you!
-----

Diagnostics
    -------
    Warning: Dropbox import not working. dropbox_key is empty.
    Info: Latest version of PHP is 8.3

    System Information
    --------------
    Lychee Version (release):                5.1.2
    DB Version:                              5.1.2
    
    composer install:                        --no-dev
    APP_ENV:                                 production
    APP_DEBUG:                               false
    APP_URL:                                 set
    APP_DIR:                                 default
    LOG_VIEWER_ENABLED:                      true
    LIVEWIRE_ENABLED:                        false
    
    System:                                  Linux
    PHP Version:                             8.2.14
    PHP User agent:                          Lychee/4 (https://lycheeorg.github.io/)
    Timezone:                                Asia/Ho_Chi_Minh
    Max uploaded file size:                  64M
    Max post size:                           128M
    Livewire chunk size:                     12.00 MB
    Max execution time:                      360
    MySQL Version:                           10.6.16-MariaDB-cll-lve
    
    exec() Available:                        yes
    Imagick Available:                       1
    Imagick Enabled:                         1
    Imagick Version:                         1809
    GD Version:                              2.3.3
    Number of foreign key:                   12 found.

    Config Information
    --------------
    version:                                 050102
    check_for_updates:                       1
    sorting_photos_col:                      taken_at
    sorting_photos_order:                    ASC
    sorting_albums_col:                      max_taken_at
    sorting_albums_order:                    ASC
    imagick:                                 1
    skip_duplicates:                         0
    small_max_width:                         0
    small_max_height:                        360
    medium_max_width:                        1920
    medium_max_height:                       1080
    lang:                                    en
    image_overlay_type:                      exif
    default_license:                         reserved
    compression_quality:                     90
    grants_full_photo_access:                1
    delete_imported:                         0
    mod_frame_enabled:                       1
    mod_frame_refresh:                       30
    thumb_2x:                                1
    small_2x:                                1
    medium_2x:                               1
    landing_page_enable:                     0
    site_owner:                              ViXiM's
    landing_title:                           ViXiM's
    landing_subtitle:                        Eat, Love, Travel...
    sm_facebook_url:                         
    sm_flickr_url:                           
    sm_twitter_url:                          
    sm_instagram_url:                        
    sm_youtube_url:                          
    landing_background:                      dist/cat.webp
    site_title:                              Eat, Love, Travel
    footer_show_copyright:                   1
    site_copyright_begin:                    2011
    site_copyright_end:                      2024
    footer_additional_text:                  Do not use if you are not permitted.
    footer_show_social_media:                0
    search_public:                           0
    SL_enable:                               0
    SL_for_admin:                            0
    recent_age:                              30
    grants_download:                         0
    photos_wraparound:                       1
    map_display:                             1
    zip64:                                   1
    map_display_public:                      1
    map_provider:                            OpenStreetMap.org
    force_32bit_ids:                         0
    map_include_subalbums:                   1
    update_check_every_days:                 3
    has_exiftool:                            0
    share_button_visible:                    0
    import_via_symlink:                      0
    has_ffmpeg:                              0
    location_decoding:                       1
    location_decoding_timeout:               30
    location_show:                           1
    location_show_public:                    0
    rss_enable:                              0
    rss_recent_days:                         7
    rss_max_items:                           100
    prefer_available_xmp_metadata:           0
    editor_enabled:                          1
    lossless_optimization:                   0
    swipe_tolerance_x:                       150
    swipe_tolerance_y:                       250
    local_takestamp_video_formats:           .avi|.mov
    log_max_num_line:                        1000
    unlock_password_photos_with_url_param:   0
    nsfw_visible:                            1
    nsfw_blur:                               0
    nsfw_warning:                            0
    nsfw_warning_admin:                      0
    nsfw_banner_override:                    
    map_display_direction:                   1
    album_subtitle_type:                     oldstyle
    upload_processing_limit:                 4
    new_photos_notification:                 0
    legacy_id_redirection:                   1
    zip_deflate_level:                       6
    SA_enabled:                              1
    default_album_protection:                1
    album_decoration:                        layers
    album_decoration_orientation:            row
    allow_username_change:                   1
    auto_fix_orientation:                    1
    use_job_queues:                          1
    random_album_id:                         starred
    use_last_modified_date_when_no_exif_date: 1
    ffmpeg_path:                             
    ffprobe_path:                            
    layout:                                  square
    date_format_photo_thumb:                 M j, Y, g:i:s A e
    date_format_photo_overlay:               M j, Y, g:i:s A e
    date_format_sidebar_uploaded:            M j, Y, g:i:s A e
    date_format_sidebar_taken_at:            M j, Y, g:i:s A e
    date_format_hero_min_max:                F Y
    date_format_hero_created_at:             M j, Y, g:i:s A T
    date_format_album_thumb:                 M Y
    upload_chunk_size:                       0
    nsfw_banner_blur_backdrop:               0
    search_pagination_limit:                 1000
    search_minimum_length_required:          4
    photo_layout_justified_row_height:       320
    photo_layout_masonry_column_width:       300
    photo_layout_grid_column_width:          250
    photo_layout_square_column_width:        200
    photo_layout_gap:                        12
    display_thumb_album_overlay:             always
    display_thumb_photo_overlay:             hover
    default_album_thumb_aspect_ratio:        1/1
    use_album_compact_header:                0
    login_button_position:                   left
    back_button_enabled:                     0
    back_button_text:                        Return to Home
    back_button_url:                         /

**Comment by ildyria** [comment, marked as answer]
From your diagnostics, your `APP_DIR` is not set. This is why your pictures are not showing.
I assume your `APP_URL` is set to `https://bachda.xyz` as we don't see warnings/errors (otherwise you would see complaints on that).

PS: thank you for opening a discussion and not an issue. <3

**Reply by vuongquoc** [reply]
Thank Mr. @ildyria ,
You always there helping me everytime...
I've fixed this problem.
Last time i install v4. i did not have to set APP_URL so my env. file don't show that instructions.
And this time, i've search and try to reinstall v5 with mysql (not sqlite3 anymore). 

It works now! Flawlessly! 

Thank you so much!

**Comment by d7415** [comment]
> Please have a look...
> https://bachda.xyz/public/

Please take the hint from https://bachda.xyz/ (or the installation instructions) and set `public` as the root.

**Reply by vuongquoc** [reply]
Thank you, @d7415
